tests/phpunit/tests/image/intermediateSize.php

```diff
@@ -91,15 +91,17 @@ static function() {
 	/**
 	 * Test that smaller sized images used when enabled.
 	 */
-	public function it_should_use_smaller_jpg_image_when_smaller_size_preferred() {
+	public function test_it_should_use_smaller_jpg_image_when_smaller_size_preferred() {
```

Comment:
Let's use a name for this test that is better aligned with core test naming patterns. They are only loosely defined, but when you look at other test names, there is usually a pattern, often being at least that the method starts with `test_{name_of_the_function}_{optionally_something_else}`. Maybe
```suggestion
	public function test_wp_image_use_alternate_mime_types_prefers_smaller_file() {
```

Reply:
Thanks for the feedback, Felix. The PR was updated with the function name change.